config/scenario_perception_pipeline_test_cases.yaml

```diff
@@ -0,0 +1,171 @@
+# test cases stored in the nav_msgs/msg/Path
+test_cases:
+  # the first test case in type of nav_msgs/msg/Path,
+  # This test case can be selected in scenario_perception_pipeline_benchmark_main node
+  # by passing selected_test_case_index argument as 0
+  - poses:
+    # one of the pose in the `poses` field of nav_msgs/msg/Path
+    - pose:
+        position:
+          x: 0.5
+          y: 0.5
+          z: 0.5
+        orientation:
+          w : 1.0
```

Comment:
I couldn't undestand why you want to use orientation in default. I have run benchmark scenarios using this test case. I will find a more convenient test case for you. But it would be so great to firstly merge this development stuff and then apply the things in this review in follow-up PRs. Does it make sense @henningkayser ?

Reply:
I was only commenting on the readability. w=1 is the identity transform assuming it is a quaternion and the other components are set to 0. It would be easier to read this file if all components have to be specified (of course this would bloat a lot of redundant content), or to leave the orientation out and implicitly default to the identity orientation. I agree that this should not block merging the PR.

Reply:
I will add the other components in follow-up PR.